modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataDefinitionResourceImpl.java

```diff
@@ -1692,8 +1693,10 @@ private void _validate(
 	private static final EntityModel _entityModel =
 		new DataDefinitionEntityModel();
 
-	@Reference
-	private DataDefinitionModelResourcePermission
+	@Reference(
+		target = "(model.class.name=com.liferay.dynamic.data.mapping.model.DataDefinition)"
+	)
```

Comment:
@MrJohn1911  use `com.liferay.data.engine.rest.dto.v2_0.DataDefinition` and change the previous changed reference to use the DTO class as well.